docs/source/Schemas.md

```diff
@@ -135,24 +135,35 @@ Both representations are binary equivalent.
 
 Arrays are currently only supported in a `struct`.
 
-### (Default) Values
+### Default, Optional and Required Values
 
-Values are a sequence of digits. Values may be optionally followed by a decimal
-point (`.`) and more digits, for float constants, or optionally prefixed by
-a `-`. Floats may also be in scientific notation; optionally ending with an `e`
-or `E`, followed by a `+` or `-` and more digits.
+There are three, mutually exclusive, reactions to the non-presence of a table's
+field in the binary data:
+
+-    Default valued fields will return the default value.
```

Comment:
nit: can we use numerical bullet points to emphasize the 3 reactions.

Reply:
Done